### PR TITLE
[openload] Added support for links starting with an underscore

### PR DESCRIPTION
**Pull request by N1k145**
Links from openload can start whith an underscore

### CODE REVIEW

**Comment by yan12125** [comment]
An example URL?

**Comment by N1k145** [comment]
For example https://openload.co/f/_-ztPaZtMhM/ but you find many more by googleing "openload.co/f/_"

**Comment by yan12125** [comment]
Good catch. Please add it to ```_TESTS``` with ```'only_matching': True```.

File: youtube_dl/extractor/openload.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 
 class OpenloadIE(InfoExtractor):
-    _VALID_URL = r'https://openload.(?:co|io)/(?:f|embed)/(?P<id>[a-zA-Z0-9-]+)'
+    _VALID_URL = r'https://openload.(?:co|io)/(?:f|embed)/(?P<id>[a-zA-Z0-9-\_]+)'
```
Comment (yan12125) [inline youtube_dl/extractor/openload.py]: _ is not a special character in Python's regular expressions. There's no need to escape it.

**Comment by TRox1972** [comment]
@N1k145 you could simplify the regular expression by using `\w-` instead

**Comment by dstftw** [comment]
Cherry-picked in 21efee5f8bc8daf0cbb5fc3408a1fc5b9d5eadcb, thanks.

**Pull request closed without merging**